pkg/scheduler/apis/config/types.go

```diff
@@ -171,6 +171,9 @@ type ElasticQuotaArgs struct {
 	// SystemQuotaGroupMax limit the maxQuota of SystemQuotaGroup
 	SystemQuotaGroupMax corev1.ResourceList
 
+	// RootQuotaGroupMax limit the maxQuota of RootQuotaGroup
+	RootQuotaGroupMax corev1.ResourceList
```

Comment:
IMO  the root-quota represents the total resources of the cluster, there should not be a separate parameter to set the default size. And what this PR expects is to expose the internal root-quota data through ElasticQuota CRD, so I understand that it is enough to modify the syncHandler in elasticquota/controller.go? Of course, there are some supporting modifications, such as the verification of webhook.

Reply:
You're right, it's not appropriate to set the max of root-quota through separate parameters. I'll make that change. However, just modifying the syncHandler in elasticquota/controller.go may not be enough to update certain data within the root-quota (e.g. request, used, runtime, etc.), because the existing code for updating used and request often "skips" the root-quota. So, we prefer to update the relevant data of the root-quota through the code we've modified. If you have any better suggestions for updating the root-quota parameters,  I am willing to consider them.

Reply:
In synchandler, count all the used and requested, and use the results as the used and requested of root-quota. The total capacity is the max of root-quota. Is this possible?

Reply:
That may be feasible, but wouldn't it look strange to update the root-quota information within the syncHandler? Why not directly update the 'used' value of root-quota in the updateGroupDeltaUsedNoLock? Wouldn't that be more logical? Our development approach is to update the relevant information of root-quota directly using the existing 'used' update method, to integrate with the existing code as much as possible and eliminate unnecessary 'skips'. Is there any specific design in the existing code development?

Reply:
fixed